src/lib/loaders/loaders_without_authentication/gravity.ts

```diff
@@ -114,7 +114,7 @@ export default (opts) => {
     >(
       ({ artist_id, partner_id }) => `partner/${partner_id}/artist/${artist_id}`
     ),
-    partnerArtistArtworksLoader: gravityLoader<
+    partnerArtistPartnerArtistArtworksLoader: gravityLoader<
```

Comment:
this is a clunky but more accurate name for this loader since it returns PartnerArtistArtworks and the other loader returns just Artworks

Reply:
possibly `partnerArtistArtworksLoader` ? (avoiding the double `partnerArtist` thing - but i hear you this naming is a bit tough and verbose)

Reply:
that was what it was originally called but the reason for the rename was because I need another loader which would more accurately be named that in the authenticated loaders:
https://github.com/artsy/metaphysics/blob/d99b86824d9ae4a07acdd95f3d58fee86f7297d4/src/lib/loaders/loaders_with_authentication/gravity.ts#L322

this loader hits this endpoint:
`partner/${partnerID}/artist/${artistID}/partner_artist_artworks`

the one I need hits this endpoint:
`partner/${partnerID}/artist/${artistID}/artworks/all`

so the name thinking was:

```
partnerArtist -> an artist managed by this partner
+ 
PartnerArtistArtworks -> return type partnerArtistArtworks
```

and the other one:

```
partnerArtist -> an artist managed by this partner
+ 
Artworks -> return type Artworks
```